vnext/JSI/Shared/ChakraRuntime.h

```diff
@@ -248,15 +239,35 @@ class ChakraRuntime : public facebook::jsi::Runtime {
   ChakraObjectRef ToChakraObjectRef(const facebook::jsi::Value &value);
   std::vector<ChakraObjectRef> ToChakraObjectRefs(const facebook::jsi::Value *value, size_t count);
 
-  // Host function and host object helpers
+  // Convenience functions for property access.
+  ChakraObjectRef GetProperty(const ChakraObjectRef &obj, const ChakraObjectRef &id);
+  inline ChakraObjectRef GetProperty(const ChakraObjectRef &obj, const char *const name) {
```

Comment:
>inline [](start = 2, length = 6)

(nit) is this needed? compilers do inlining anyways

Reply:
Yea the compiler should automatically do inlining here, but I like the explicitness.

---
In reply to: [341261679](https://github.com/microsoft/react-native-windows/pull/3568#discussion_r341261679) [](ancestors = 341261679)